### DISCUSSION TITLE
Default cake script name build.cake autocomplete collides with build.ps1

### DISCUSSION THREAD

**Discussion by nmbro** [cake-build/cake#3162] [Ideas]
Hi

I have a tiny inconvenience, whenever I use the Cake Command in vscode to install the bootstrapper script.

The default bootstrapper cake script name is build.cake - meaning that if you go into your repository and expect build\<TAB> to autocomplete into build.ps1 you are going to have a bad time, because the first build\<TAB> autocompletes to build.cake, which ofc. isn't an executable binary on Windows.

I suggest that we change the bootstrap script default script name to setup.cake or something that isn't build.cake

**Comment by kjjuno** [comment]
I have run into this same issue. I find myself frequently renaming the bootstrapper to something like `make.ps1` to avoid having to tab multiple times

**Comment by nmbro** [comment]
yeah - but the default name of the bootstrapper is build.ps1 - and the default templated build script is build.cake.

IMO it should would out of the box - no renamed required - no bootstrap file editing needed - it just a better experience.
But ofc. if users have other local files/folders that conflict with the build\<TAB> autocomplete it's on them - but out of the box Cake shouldn't create this dilemma to begin with.

**Comment by gitfool** [comment]
I use `cake.ps1` and `cake.sh`; coming from using `rake` and `make`, that made the most sense to me.

**Comment by devlead** [comment]
@gitfool advantage of renaming bootstrappers is that it won't break existing builds. While changing implicit default script from build.cake will,  as people update their Cake version. Bootstrappers in general checked into repo and only changed when people explicitly chooses to do so.
